src/pages/messages/messages.ts

```diff
@@ -237,6 +255,16 @@ export class MessagesPage implements OnInit, OnDestroy {
     return autoScroller;
   }
 
+  getLocation(locationString: string): Location {
+    const splitted = locationString.split(',');
+
+    return <Location>{
+      lat: Number(splitted[0]),
+      lng: Number(splitted[1]),
+      zoom: Math.min(Number(splitted[2] || 0), 19)
```

Comment:
After you apply the fix above you don't need to convert to numbers or use the `|| 0` notation